spec/webauthn/attestation_statement/android_safetynet_spec.rb

```diff
@@ -84,15 +86,35 @@
       let(:nonce) { Base64.strict_encode64(OpenSSL::Digest::SHA256.digest("something else")) }
 
       it "returns false" do
-        expect(statement.valid?(authenticator_data, client_data_hash)).to be_falsy
+        expect(statement.valid?(authenticator_data, client_data_hash, trust_store: trust_store)).to be_falsy
       end
     end
 
     context "when ctsProfileMatch is not true" do
       let(:cts_profile_match) { false }
 
       it "returns false" do
-        expect(statement.valid?(authenticator_data, client_data_hash)).to be_falsy
+        expect(statement.valid?(authenticator_data, client_data_hash, trust_store: trust_store)).to be_falsy
+      end
+    end
+
+    context "when timestampMs is set to future" do
+      let(:timestamp) { Time.now.to_i + 60 }
+
+      it "returns false" do
+        Timecop.freeze(Time.now) do
+          expect(statement.valid?(authenticator_data, client_data_hash, trust_store: trust_store)).to be_falsy
+        end
+      end
+    end
+
+    context "when timestampMs is older than one minute" do
+      let(:timestamp) { Time.now.to_i - 61 }
+
+      it "returns false" do
+        Timecop.freeze(Time.now) do
+          expect(statement.valid?(authenticator_data, client_data_hash, trust_store: trust_store)).to be_falsy
+        end
```

Comment:
Should we move these test cases to `spec/android_safetynet/attestation_response_spec.rb` after we moved the validation there?